src/applications/proxy-rewrite/redirects/disabilityRedirects.json

```diff
@@ -1,4 +1,79 @@
 [
+  {
+    "domain": "www.cem.va.gov",
+    "src": "/cem/burial_benefits/eligible.asp",
+    "dest": "www.va.gov/burials-memorials/eligibility/"
+  },
+  {
+    "domain": "www.cem.va.gov",
+    "src": "/cem/Women_and_Minority_Veterans.asp",
+    "dest": "/burials-memorials/eligibility/"
+  },
+  {
+    "domain": "www.cem.va.gov",
+    "src": "/cem/burial_benefits/burial_at_sea.asp",
+    "dest": "/burials-memorials/eligibility/burial-at-sea/"
+  },
+  {
+    "domain": "www.cem.va.gov",
+    "src": "/cem/burial_benefits/private_cemetery.asp",
+    "dest": "/burials-memorials/eligibility/burial-in-private-cemetery/"
+  },
+  {
+    "domain": "www.cem.va.gov",
+    "src": "/cem/burial_benefits/burial_flags.asp",
+    "dest": "/burials-memorials/memorial-items/burial-flags/"
+  },
+  {
+    "domain": "www.cem.va.gov",
+    "src": "/cem/hmm/index.asp",
+    "dest": "/burials-memorials/memorial-items/headstones-markers-medallions/"
+  },
+  {
+    "domain": "www.cem.va.gov",
+    "src": "/cem/hmm/eligibility.asp",
+    "dest": "/burials-memorials/memorial-items/headstones-markers-medallions/"
+  },
+  {
+    "domain": "www.cem.va.gov",
+    "src": "/cem/hmm/order_instructions.asp",
+    "dest": "/burials-memorials/memorial-items/headstones-markers-medallions/"
+  },
+  {
+    "domain": "www.cem.va.gov",
+    "src": "/cem/hmm/apply_by_fax.asp",
+    "dest": "/burials-memorials/memorial-items/headstones-markers-medallions/"
+  },
+  {
+    "domain": "www.cem.va.gov",
+    "src": "/cem/hmm/order_by_mail.asp",
+    "dest": "/burials-memorials/memorial-items/headstones-markers-medallions/"
+  },
+  {
+    "domain": "www.cem.va.gov",
+    "src": "/cem/pmc.asp",
+    "dest": "/burials-memorials/memorial-items/presidential-memorial-certificates/"
+  },
+  {
+    "domain": "www.cem.va.gov",
+    "src": "/cem/pre-need/index.asp",
+    "dest": "/burials-memorials/pre-need-eligibility/"
+  },
+  {
+    "domain": "www.cem.va.gov",
+    "src": "/CEM/pre-need/FAQ/",
```

Comment:
This is the only one I'm worried about due to capital case 😬

Reply:
It converts to lowercase before comparing with the window location 🙂

Reply:
Wonderful, thank you for confirming! 💯